crates/bevy_window/src/window.rs

```diff
@@ -5,6 +5,26 @@ use raw_window_handle::RawWindowHandle;
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
 pub struct WindowId(Uuid);
 
+/// Behavior of the presentation engine based on frame rate.
+#[repr(C)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
+pub enum PresentMode {
```

Comment:
New users are definitely going to be confused by this name.

Can we use `PresentationMode` instead? Are there any other more approachable synonyms?

Reply:
This is the name used by `wgpu`. It seems that other wgpu items are used or mirrored directly, or given a `Wgpu` prefix.

https://docs.rs/wgpu-types/0.12.0/wgpu_types/enum.PresentMode.html

Reply:
I don't think a new user would be any less confused by the term `VSync`. I think it's preferable to keep the names aligned with the corresponding items in WGPU and the Vulkan spec.

Reply:
Could you add a doc alias to vsync to help transition, and maybe add a link in the doc comments to where its use, in `set_present_mode` and the `WindowDescriptor` struct?